l1-contracts/scripts/utils.ts

```diff
@@ -10,6 +10,9 @@ const warning = chalk.bold.yellow;
 export const L1_TO_L2_ALIAS_OFFSET = "0x1111000000000000000000000000000000001111";
 export const GAS_MULTIPLIER = 1;
 
+// Bit shift by 32 does not work in JS, so we have to multiply by 2^32
+const SEMVER_MINOR_VERSION_MULTIPLIER = 4294967296;
```

Comment:
Should you use big number, or this is safe?..

Reply:
safe up to 2^53